RELEASE/scripts/sl_ascend/sl_cooking.ash

```diff
@@ -72,7 +72,7 @@ float expectedAdventuresFrom(item it)
 	if(it == $item[magical sausage]) return 1;
 
 	if ($items[campfire baked potato, campfire beans, campfire coffee, campfire hot dog, campfire s\'more, campfire stew] contains it) {
-		return 4.5; // I guess?
+		return 3.5; // I guess?
```

Comment:
Actually I should have caught this before, but aren't the 1 stick foods 3.5 and the 2 stick foods 4.5?

Reply:
I think you're right! For now I'll switch over to only consuming the 2-fullness items, since the knapsack solver doesn't really handle "there are lots of equally good consumables made from this list of ingredients" particularly well.

Reply:
They're all 1-fullness, some of them just take 2 sticks to make instead of 1